### PR TITLE
Tranforming data example

### PR DESCRIPTION
**Pull request by K4liber**

### CODE REVIEW

File: TranformingData/transformingDataExample.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        "rError", # Difference beetwen source point and recontructed point
+        "volD",   # Volumes indexes difference
+        "lorL",   # LOR length
+        "deg3D",  # Angle beetwen lines (in XYZ geometry) connecting detection points with the center of detector
+        "deg2D",  # Angle beetwen lines (in XY geometry) connecting detection points with the center of detector
+        "rL",     # Distance beetween reconstructed point and the center of detector
+        "eSum"    # Sum of the detecions energies
+    ]
+
+pathToFile = '/mnt/home/jbielecki1/NEMA/NEMA_IQ_384str_N0_1000_COINCIDENCES_PREPARED_part00'
```
Comment (graag) [inline TranformingData/transformingDataExample.py]: Better default would be working directory ...

File: TranformingData/transformingDataExample.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3.6
+
+import dask.dataframe as dd
+import pandas as pd
+import sys
+import pickle
+import matplotlib.pyplot as plt
+
+def dataFrameNames():
```
Comment (graag) [inline TranformingData/transformingDataExample.py]: Why are you using a function for this?

Wouldn't global variable be enough?

File: README.md [hunk 3]
```diff
@@ -18,8 +18,8 @@ You can change it inside the script.
 **Input columns**: Discribed in the function *dataFrameNames()* at */TransformingData/transformingData.py* script  
 **Output format**: binary file  
 **Output columns**: same as above + result of the function *featureEngineering()* at */TransformingData/transformingData.py* script    
-**Purpose**: Feature engineering and saving as binary file  
+**Purpose**: Feature engineering - add extra features (listed in return of the *featureEngineering(row)* function in /TransformingData/transformingData.py script) to the original ones (listed in *dataFrameNames()* function in /TransformingData/transformingData.py script). After the transformation new data is saved as binary file (serialization made using the *Pickle* package).
```
Comment (graag) [inline README.md]: There is no information that what is actually pickled are pandas DataFrames.

While I agree that putting exact description of the format in the README will at some point mean that it is outdated, I do not like how it is defined all over the transformingData.py

I would propose to have two globals at the beginning of the script that define both original and new attributes. With a nice comment to easily distinguish which is which.

The README could point to those globals as it currently does. Still it should better describe the format as pandas DataFrames

**Review by graag — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (wkrzemien) [reply README.md]: @K4liber Please answer and/or add modification to your code

File: TranformingData/transformingDataExample.py (hunk 1 above)
Comment (K4liber) [reply TranformingData/transformingDataExample.py]: https://github.com/JPETTomography/j-pet-multiphoton-classification/pull/2/commits/6ae9978ab7b2844a0dedb0df19e71f88f03c0797

File: TranformingData/transformingDataExample.py (hunk 2 above)
Comment (K4liber) [reply TranformingData/transformingDataExample.py]: https://github.com/JPETTomography/j-pet-multiphoton-classification/pull/2/commits/8bc76cf6252e9c9880dde9f18f530bee3f2c9c38

Comment (K4liber) [reply TranformingData/transformingDataExample.py]: There was a reason that I truly don't remember.
https://github.com/JPETTomography/j-pet-multiphoton-classification/pull/2/commits/c6d4263dfb9ce1c0335812bff7ab0f0074c53b53

File: README.md (hunk 3 above)
Comment (K4liber) [reply README.md]: https://github.com/JPETTomography/j-pet-multiphoton-classification/pull/2/commits/c6d4263dfb9ce1c0335812bff7ab0f0074c53b53

File: TranformingData/transformingDataExample.py (hunk 2 above)
Comment (K4liber) [reply TranformingData/transformingDataExample.py]: I've remembered. I was using it in another file (in another script) and imported names as function cause I cant import variable ;) This not make sense in this case.

**Pull request merged**